docs/programmers_guide/guide.md

```diff
@@ -552,11 +552,10 @@ the period of time including the transaction.
 
 [See more about blocks and transactions processing here](../../eth/stagedsync/README.md)
 
-
 Dev Net with Geth nodes
 -----------------------
 
-- create somewhere file `erigon.dev` with content `26e86e45f6fc45ec6e2ecd128cec80fa1d1505e5507dcd2ae58c3130a7a97b48`
-- run: `geth account import ./erigon.dev`
-- add to geth flag `--miner.etherbase=67b1d87101671b127f5f8714789c7192f7ad340e`
-- follow https://geth.ethereum.org/docs/interface/private-network
+* create somewhere file `erigon.dev` with content `26e86e45f6fc45ec6e2ecd128cec80fa1d1505e5507dcd2ae58c3130a7a97b48`
```

Comment:
This change was done by the markdownlint extension, to keep list indentation symbol consistent in the same file.